src/scripts/google-analytics.js

```diff
@@ -37,3 +37,9 @@ module.exports.track = function track(...args) {
     window.ga(`send`, ...args)
   }
 }
+
+module.exports.setGoogleAnalyticsPage = function track(...args) {
+  if (window.ga) {
+    window.ga(`set`, `page`, ...args)
```

Comment:
will this `send` the event or just set the current page?

Reply:
good call. upon further investigation, `send` is required.